jobrunner/controller/main.py

```diff
@@ -533,7 +533,7 @@ def update_job(job):
     update(job, exclude_fields=["cancelled"])
 
 
-def create_task_for_job(job):
+def create_task_for_job(job, **kwargs):
```

Comment:
A micro thing, but I'm not sure it makes sense to be changing the prod code here just to make test setup more convenient. Given that this function returns the object and doesn't insert it, can't the factory function modify it afterwards to apply the `kwargs`?

Reply:
Yeah, it can.

I originally tried using the whole create-then-insert-in-transaction logic, which made that harder. When I went to just using create_task_for_job, then I missed that that could work.